docs/source/index.rst

```diff
@@ -66,9 +66,9 @@ When the ``.fit()`` method is called, it automatically removes any examples iden
    from sklearn.linear_model import LogisticRegression
    from cleanlab.classification import CleanLearning
 
-   clf = LogisticRegression() # any classifier implementing the sklearn API
+   clf = LogisticRegression() # Here we've used sklearn's Logistic Regression model, but this can be any classifier that implements sklearn's API.
    cl = CleanLearning(clf=clf)
-   cl.fit(X=X, labels=labels)
+   cl.fit(X=X, labels=y)
```

Comment:
I may be out of context here, but I had changed `labels` to `y` universally, even in examples, everywhere. Any reason to make it `y` here?

Reply:
Good catch. This is actually wrong in `master` right now, and I fixed it in one commit in this PR, and then reverted it back to master accidentally. Fixed.